xml/System.Net/ServicePointManager.xml

```diff
@@ -804,6 +804,12 @@ For versions of the .NET Framework through the .NET Framework 4.6.2, no default
  An application can set the <xref:System.Net.ServicePointManager.ServerCertificateValidationCallback%2A> property to a method to use for custom validation by the client of the server certificate. When doing custom validation, the `sender` parameter passed to the <xref:System.Net.Security.RemoteCertificateValidationCallback> can be a host string name or an object derived from <xref:System.Net.WebRequest> (<xref:System.Net.HttpWebRequest>, for example) depending on the <xref:System.Net.ServicePointManager.CertificatePolicy%2A> property.  
   
  When custom validation is not used, the certificate name is compared with host name used to create the request. For example, if <xref:System.Net.WebRequest.Create%28System.String%29> was passed a parameter of "https://www.contoso.com/default.hmtl", the default behavior is for the client to check the certificate against www.contoso.com.  
+
+ Despite being a multicast delegate, only the value returned from the last-executed event handler is considered authoritive. 
+
+ In combination with the <xref:System.Net.ServicePointManager%2A> being a static shared across threads, the callback is an attack vector for malicious
```

Comment:
Please change this to:
```markdown
As <xref:System.Net.ServicePointManager> is shared across threads, malicious code 
within your application can override the settings you have made. 
Always use code, libraries, and NuGet packages that you trust in your project.
```

Reply:
Thanks for considering the text. It opened my eyes that in fact the problem is present for both consuming as well as embedded components. 
As a vendor of a sql engine component we were worried about another thread from embedding code circumventing our code is integrity but of course it also holds that the embedding code can be compromised by a fraudulent component. 
We have tested that through separated app domains it can be secured with overhead. But luckily there is no app domain creation event that can be hooked into outside of the clr. We had a similar issue with dependency injection. Luckily most people can not easily create a clr of their own. 

I would like to propose a slightly modified version of the text: that both components being embedded should be considered for trust but also the embedding code. 

Is that a good adaption?